server/game/basecard.js

```diff
@@ -366,8 +375,8 @@ class BaseCard {
         return this.keywords.getValues();
     }
 
-    hasPrintedKeyword(keyword) {
-        return this.printedKeywords.includes(keyword.toLowerCase());
+    hasPrintedKeyword(keyword, ignoreValue = false) {
+        return this.printedKeywords.some(printedKeyword => ignoreValue ? printedKeyword.includes(keyword) : printedKeyword === keyword);
```

Comment:
As best as I can tell, this new parameter is never actually used. (And I would prefer a different method rather than this "ignoreValue" boolean if it were)

Reply:
Yep, leftover code from when **Doran Martell (AHaH)** was doing something whacky by giving all cards in shadows `Shadow (3)` rather than `Shadow (X)`.

Regardless, reverted in 31fa026dc6b1e7e530b9fea7fef084789d887b57

Can mark as resolved.